sdk/maps/azure-maps-timezone/azure/maps/timezone/models/__init__.py

```diff
@@ -0,0 +1,14 @@
+# coding=utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ._models import (
+    LatLon
+)
+
+__all__ = [
+    'LatLon'
+]
```

Comment:
Are we going to just expose `LatLon` only? If you want to expose other models/return object from generated files, you can add them here. If not, I would recommend flattening those to be more self-explanatory